src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -152,20 +158,31 @@ class ActivitySidebar extends React.PureComponent<Props, State> {
     constructor(props: Props) {
         super(props);
         // eslint-disable-next-line react/prop-types
-        const { logger } = this.props;
+        const { activeFeedEntryId, activeFeedEntryType, logger } = this.props;
 
         mark(MARK_NAME_DATA_LOADING);
 
         logger.onReadyMetric({
             endMarkName: MARK_NAME_JS_READY,
         });
-        this.state = {};
+        this.state = { activeFeedEntryId, activeFeedEntryType, approverSelectorContacts: [] };
```

Comment:
We should almost always avoid duplicating our props or any prop-derived value in state, as it can easily lead to synchronization issues. Let's update the logic to use the values directly from props.

Reply:
I understand your point but in this case I don't see any alternative really. The `activeFeedEntryId` and `activeFeedEntryType` props represent the values of url params. Previously they were simply passed to `ActivityFeed`, where logic for displaying "selected state" lives.

Now with the possibility of a reply id being set as `activeFeedEntryId` we need to check if it is somewhere within feed items. If it turns out that it's a reply we need to pass its parent as `activeFeedEntryId` as the parent needs to be "selected/highlighted". In that case I'm updating values in state and pass them to `ActivityFeed`. If no action is taken. Moreover if the reply id is not found within feed items, we need to fetch for that reply's data in order to find its parent and then set its id in state. 

Ultimately this is a temporary solution until we get the `partentId` as an url param. With that we would not need to store the active item id in state as it would simply be either the `itemId` or the `parentId`.